packages/backend/src/api/request.ts

```diff
@@ -72,13 +73,9 @@ export function buildRequest(options: CreateBackendApiOptions) {
     } = options;
     const { path, method, queryParams, headerParams, bodyParams } = requestOptions;
 
-    const key = secretKey || apiKey;
+    const key = secretKey || apiKey || '';
```

Comment:
Why do we need to fallback to ''?

Reply:
We can omit it if the validation function can receive undefined.